assets/js/script.js

```diff
@@ -1,57 +1,47 @@
 var global = {};
 var formHandlerEl = function (event) {
+    console.log("hello");
     event.preventDefault();
     var cityStateInputEl = document.getElementById("location-input").value;
     // run api call next
 
     var parksApiUrl = "https://developer.nps.gov/api/v1/parks?limit=50&start=0&q=" + cityStateInputEl + "&api_key=xG0N7G0NIR00rad6uzGstrePJgkPJ12OyeOMTr9q";
-    var foodApiUrl = "https://api.documenu.com/v2/restaurants/search/fields?state=" + cityStateInputEl + "&key=0d461c352166be6cd4a1a1e0925996b4";
 
-    //grabbing users search input
-    var searchLocation = document.getElementById("location-input").value;
-    console.log(searchLocation);
+
     //grabbing the state of the checkboxes no matter if they are checked or unchecked
     var parkCheck = document.getElementById("park-checkmark").checked;
     var diningCheck = document.getElementById("dining-checkmark").checked;
     var lodgingCheck = document.getElementById("lodging-checkmark").checked;
-    console.log(parkCheck);
-    console.log(diningCheck);
-    console.log(lodgingCheck);
 
     fetch(parksApiUrl).then(function (response) {
         if (response.ok) {
             response.json().then(function (data) {
                 console.log(data.data);
-                for (i = 0; i < 3; i++) {
+                for (i = 0; i < 30; i++) {
                     var name = data.data[i].name;
                     global.zipCode = data.data[i].addresses[0].postalCode;
                     var descrip = data.data[i].description;
 
                     var parkNames = document.getElementById("park-list");
-                    var parkNameList = document.createElement("div");
+                    var parkNameList = document.createElement("li");
+                    parkNameList.setAttribute("class", "text-xl")
+                    parkNameList.setAttribute("class", "font-bold");
                     parkNameList.textContent = name;
                     parkNames.appendChild(parkNameList);
 
                     var parkLinks = document.createElement("p");
-                    parkLinks.innerHTML = "<button data-zipcode = '" + global.zipCode + "'>Click me</button>";
+                    parkLinks.innerHTML = "<button data-zipcode = '" + global.zipCode + "'>Click To See Nearby Dining</button>";
                     parkNames.appendChild(parkLinks);
                    
+                    var parksDescrip = document.createElement("p");
+                    parksDescrip.innerText = descrip;
+                    parkNames.appendChild(parksDescrip);
                 
 
                 }
             })
         }
     });
-
-    fetch(foodApiUrl).then(function(response) {
```

Comment:
@LennonSimpson Why did you delete this?  Did you mean to put it back and forget?